posydon/binary_evol/simulationproperties.py

```diff
@@ -255,13 +253,21 @@ def pre_step(self, binary, step_name):
 
     def post_step(self, binary, step_name):
         """Record the duration of the step."""
+
+        ## do not record redirect step times if history_verbose=False
+        if not binary.history_verbose and binary.event is not None:
+            if "redirect" in binary.event:
+                return binary
```

Comment:
I wondering, whether it is the right place here. Because it requires to repeat it for every hook.
The other option would be to put it up one level, and apply it before the `post_step` function of any gets called.

Reply:
I would think we don't want to call this before every hook, because hooks could be doing anything. For example, what if someone creates a hook meant to do something with the binary _only_ for redirect steps?

Reply:
If keeping it to be added to any hook, we should get a documentation/instruction for this, so that everyone writing a hook knows about.

Reply:
Ok, from that perspective I think your suggested change makes sense. I think that the instances where people will create hooks completely independent of the history will be rare.